content/applications/inventory_and_mrp/manufacturing/basic_setup/mo_costs.rst

```diff
@@ -0,0 +1,201 @@
+same costs. This is the *estimated* cost of completing the |MO|.
+
+However, once work commences, the values in the :guilabel:`Real Cost` column may begin to diverge
+from the ones in the :guilabel:`MO Cost` column. This happens if a different component quantity is
+used than was listed on the |MO|, or if the duration of a work order is shorter or longer than
+expected.
+
+Once the |MO| has been completed by clicking :guilabel:`Produce All`, the values in the
+:guilabel:`MO Cost` column update to match those displayed in the :guilabel:`Real Cost` column.
+
+.. image:: mo_costs/overview.png
+   :align: center
+   :alt: The MO Overview page.
+
+Average manufacturing cost
+==========================
+
+In addition to the cost of each individual |MO| for a product, Odoo also tracks the average cost of
+manufacturing the product, taking into account the cost of every completed |MO|. To view this,
+navigate to :menuselection:`Inventory app --> Products --> Products`, and select a product.
+
+The manufacturing cost of the product is displayed per unit of measure in the :guilabel:`Cost`
+field. The value continues to update as the costs of additional |MOs| are factored into the average
+cost.
+
+To the right of the :guilabel:`Cost` field is a :guilabel:`Compute Price from BoM` button. Click
+this button to reset the cost of the product to the expected cost, which only takes into account the
+components and operations listed on the |BoM|.
+
+Be aware that clicking :guilabel:`Compute Price from BoM` does not set the price permanently. The
+cost continues to update based on the average of the |BoM| price and the real cost of any future
+|MOs|.
+
+.. admonition:: Example workflow: manufacturing cost
+   :class: alert alert-success
+
+   Golf product manufacturer *Fairway Fields* produces a variety of golf products, include an indoor
+   *putting green*. They have configured a |BoM| for the putting green so that Odoo automatically
+   calculates the manufacturing cost of each putting green |MO|.
+
+   The |BoM| lists two components:
+
+   - One unit of *green felt*, which costs $20.00.
+   - One unit of a *rubber pad*, which costs $30.00.
+
+   The |BoM| also lists four operations, all of which are carried out at *Assembly Station 1*, which
+   has an hourly operating cost of $30.00:
+
+   - *Cut felt*, which has a default duration of seven minutes, for a total cost of $3.50.
+   - *Cut rubber pad*, which has a default duration of five minutes, for a total cost of $2.50.
+   - *Attach pad to felt*, which has a default duration of 15 minutes, for a total cost of $7.50.
+   - *Cut holes*, which has a default duration of three minutes, for a total cost of $1.50.
+
+   Altogether, the components required to produce one putting green cost $50.00, and the operations
+   required cost $15.00, for a total manufacturing cost of $65.00. This cost is reflected in the
+   Cost field on the putting green's product form.
```

Comment:
I feel like "cost" should have a GUI? If not, I don't think it needs to be capitalized.